docs/k8s-quickstart.asciidoc

```diff
@@ -263,15 +264,10 @@ EOF
 ----
 
 [float]
-[id="{p}-deep-dive"]
-=== Deep dive
+[id="{p}-persistent-storage"]
+=== Use persistent storage
 
-Now that you have completed the quickstart, you can try out more features like using persistent storage.
-
-[float]
-==== Use persistent storage
-
-The cluster that you deployed in this quickstart uses an link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[emptyDir volume], which might not qualify for production workloads.
+Now that you have completed the quickstart, you can try out more features like using persistent storage. The cluster that you deployed in this quickstart uses an link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[emptyDir volume], which might not qualify for production workloads.
```

Comment:
Actually we don't default to `emptyDir` anymore in recent master, so that needs updating as well. We now add a persistent volume claim per default of 1GiB and without a storage classs set which means that whatever default storage class is defined in the k8s cluster will be provisioned.